src/interp/interp.cc

```diff
@@ -2060,17 +2091,48 @@ RunResult Thread::DoSimdReplace(Instr instr) {
   return RunResult::Ok;
 }
 
-template <typename T> struct Simd128;
```

Comment:
Wrap this block in off?

Reply:
Wrapped... But I can't help feeling we are adding too many of this exceptions though. If we strongly want single lines to be preserved, I actually think turning on those `AllowShort***` options might be better (it also changes bunch of other existing code though)

Reply:
I think the rules are correct, but maybe be want exceptions (perhaps many) in this codebase?    Perhaps we not ready for rigid enforcement in wabt yet?